src/main/java/net/snowflake/ingest/utils/ParameterProvider.java

```diff
@@ -46,6 +46,9 @@ public class ParameterProvider {
   public static final String BDEC_PARQUET_COMPRESSION_ALGORITHM =
       "BDEC_PARQUET_COMPRESSION_ALGORITHM".toLowerCase();
 
+  public static final String ENABLE_NEW_JSON_PARSING_LOGIC =
```

Comment:
I don't think we need this parameter control, if this is to prevent regression, customer can go back to the previous version. 

The goal of this class is not for enable/disable parameter control, 99% of the parameters in the class are added because not single value would work for all cases.

Reply:
Discuss with @sfc-gh-tzhang offline, for this PR we will keep the parameter there, I created SNOW-1646229 for a cleanup.

Reply:
Could you please add the TODO with JIRA number in the comment?